tests/ui/intern-browserstack.js

```diff
@@ -3,6 +3,7 @@
 // packages, suites, excludeInstrumentation, and (if you want functional tests) functionalSuites.
 define(['./_base', './_cli', 'intern'], function(config, cli, intern) {
 
+    // Browsers to run integration testing against.
```

Comment:
Also, if we leave off the version here, BrowserStack says it will use the latest stable version of the browser available on the platform. Should we just leave it off?